src/lib/ursadb.py

```diff
@@ -43,16 +42,17 @@ def make_socket(self, recv_timeout: int = 2000) -> zmq.Context:
     def query(
         self,
         query: str,
-        taint: Optional[str] = None,
+        taints: List[str] = None,
```

Comment:
default value is not of type `List[str]`

```suggestion
        taints: List[str] = [],
```